packages/portal-proto/src/features/facets/EnumFacet.tsx

```diff
@@ -397,7 +403,6 @@ const EnumFacet: React.FC<FacetCardProps<EnumFacetHooks>> = ({
                                   e.currentTarget.checked,
                                 )
                               }
-                              aria-label={`checkbox for ${field}`}
```

Comment:
don't need to create a separate hidden label you can use area-label, can just be aria-label={value}

Reply:
using just aria-label now